UEBPrep/app/models/card.rb

```diff
@@ -21,6 +21,8 @@ def get_content
           @content = TextContent.find(self.content_id)
         when 2
           @content = TitledCardContent.find(self.content_id)
+        when 3
+          @content = ImageCardContent.find(self.content_id)
```

Comment:
yup I'm going to have to update the card model method I made for update to include this new content type as well. Hopefully this is the last major type of card we have to do, otherwise we are going to have to find a smoother way to do this.